pom.xml

```diff
@@ -38,12 +38,12 @@ THE SOFTWARE.
 		This plugin sends a http request to an url with some parameters.
 		See the release notes for changes and compatibility https://github.com/jenkinsci/http-request-plugin/releases
 	</description>
-	<url>http://wiki.jenkins-ci.org/display/JENKINS/HTTP+Request+Plugin</url>
+	<url>https://wiki.jenkins.io/display/JENKINS/HTTP+Request+Plugin</url>
```

Comment:
Switch to GitHub maybe? https://groups.google.com/forum/#!topic/jenkinsci-dev/VSdfVMDIW-A

Reply:
That's up to the plugin maintainer.

I suggest to perform such a change in a dedicated pull request as it also requires updates to the README and Confluence page.

Reply:
I like the idea of GitHub doc, but we can do it in an specific PR